.github/workflows/versionincrement.yaml

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
The files in this folder are .yml, not .yaml, rename.

Reply:
Replaced with yml db.